crates/sui-framework/packages/sui-framework/sources/coin.move

```diff
@@ -162,6 +162,21 @@ public fun put<T>(balance: &mut Balance<T>, coin: Coin<T>) {
     balance.join(into_balance(coin));
 }
 
+// === Address Balance <-> Coin utility functions ===
+
+/// Redeem a `Withdrawal<Balance<T>>` and create a `Coin<T>` from the withdrawn Balance<T>.
+public fun redeem_funds<T>(
+    withdrawal: sui::funds_accumulator::Withdrawal<Balance<T>>,
+    ctx: &mut TxContext,
+): Coin<T> {
+    from_balance(withdrawal.redeem(), ctx)
```

Comment:
The inner function here will have private generics, you cannot call it directly.
```suggestion
    balance::redeem_funds(withdrawal, ctx).into_coin()
```